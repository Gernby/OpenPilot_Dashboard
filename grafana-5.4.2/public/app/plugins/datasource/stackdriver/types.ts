export enum MetricFindQueryTypes {
  Services = 'services',
  MetricTypes = 'metricTypes',
  LabelKeys = 'labelKeys',
  LabelValues = 'labelValues',
  ResourceTypes = 'resourceTypes',
  Aggregations = 'aggregations',
  Aligners = 'aligners',
  AlignmentPeriods = 'alignmentPeriods',
}

export interface VariableQueryData {
  selectedQueryType: string;
  metricDescriptors: any[];
  selectedService: string;
  selectedMetricType: string;
  labels: string[];
  labelKey: string;
  metricTypes: Array<{ value: string; name: string }>;
  services: Array<{ value: string; name: string }>;
}
