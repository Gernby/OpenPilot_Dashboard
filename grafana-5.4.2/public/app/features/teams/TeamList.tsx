import React, { PureComponent } from 'react';
import { connect } from 'react-redux';
import { hot } from 'react-hot-loader';
import PageHeader from 'app/core/components/PageHeader/PageHeader';
import DeleteButton from 'app/core/components/DeleteButton/DeleteButton';
import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
import PageLoader from 'app/core/components/PageLoader/PageLoader';
import { NavModel, Team } from '../../types';
import { loadTeams, deleteTeam, setSearchQuery } from './state/actions';
import { getSearchQuery, getTeams, getTeamsCount } from './state/selectors';
import { getNavModel } from 'app/core/selectors/navModel';

export interface Props {
  navModel: NavModel;
  teams: Team[];
  searchQuery: string;
  teamsCount: number;
  hasFetched: boolean;
  loadTeams: typeof loadTeams;
  deleteTeam: typeof deleteTeam;
  setSearchQuery: typeof setSearchQuery;
}

export class TeamList extends PureComponent<Props, any> {
  componentDidMount() {
    this.fetchTeams();
  }

  async fetchTeams() {
    await this.props.loadTeams();
  }

  deleteTeam = (team: Team) => {
    this.props.deleteTeam(team.id);
  };

  onSearchQueryChange = event => {
    this.props.setSearchQuery(event.target.value);
  };

  renderTeam(team: Team) {
    const teamUrl = `org/teams/edit/${team.id}`;

    return (
      <tr key={team.id}>
        <td className="width-4 text-center link-td">
          <a href={teamUrl}>
            <img className="filter-table__avatar" src={team.avatarUrl} />
          </a>
        </td>
        <td className="link-td">
          <a href={teamUrl}>{team.name}</a>
        </td>
        <td className="link-td">
          <a href={teamUrl}>{team.email}</a>
        </td>
        <td className="link-td">
          <a href={teamUrl}>{team.memberCount}</a>
        </td>
        <td className="text-right">
          <DeleteButton onConfirmDelete={() => this.deleteTeam(team)} />
        </td>
      </tr>
    );
  }

  renderEmptyList() {
    return (
      <div className="page-container page-body">
        <EmptyListCTA
          model={{
            title: "You haven't created any teams yet.",
            buttonIcon: 'fa fa-plus',
            buttonLink: 'org/teams/new',
            buttonTitle: ' New team',
            proTip: 'Assign folder and dashboard permissions to teams instead of users to ease administration.',
            proTipLink: '',
            proTipLinkTitle: '',
            proTipTarget: '_blank',
          }}
        />
      </div>
    );
  }

  renderTeamList() {
    const { teams, searchQuery } = this.props;

    return (
      <div className="page-container page-body">
        <div className="page-action-bar">
          <div className="gf-form gf-form--grow">
            <label className="gf-form--has-input-icon gf-form--grow">
              <input
                type="text"
                className="gf-form-input"
                placeholder="Search teams"
                value={searchQuery}
                onChange={this.onSearchQueryChange}
              />
              <i className="gf-form-input-icon fa fa-search" />
            </label>
          </div>

          <div className="page-action-bar__spacer" />

          <a className="btn btn-success" href="org/teams/new">
            New team
          </a>
        </div>

        <div className="admin-list-table">
          <table className="filter-table filter-table--hover form-inline">
            <thead>
              <tr>
                <th />
                <th>Name</th>
                <th>Email</th>
                <th>Members</th>
                <th style={{ width: '1%' }} />
              </tr>
            </thead>
            <tbody>{teams.map(team => this.renderTeam(team))}</tbody>
          </table>
        </div>
      </div>
    );
  }

  renderList() {
    const { teamsCount } = this.props;

    if (teamsCount > 0) {
      return this.renderTeamList();
    } else {
      return this.renderEmptyList();
    }
  }

  render() {
    const { hasFetched, navModel } = this.props;

    return (
      <div>
        <PageHeader model={navModel} />
        {hasFetched ? this.renderList() : <PageLoader pageName="Teams" />}
      </div>
    );
  }
}

function mapStateToProps(state) {
  return {
    navModel: getNavModel(state.navIndex, 'teams'),
    teams: getTeams(state.teams),
    searchQuery: getSearchQuery(state.teams),
    teamsCount: getTeamsCount(state.teams),
    hasFetched: state.teams.hasFetched,
  };
}

const mapDispatchToProps = {
  loadTeams,
  deleteTeam,
  setSearchQuery,
};

export default hot(module)(connect(mapStateToProps, mapDispatchToProps)(TeamList));
