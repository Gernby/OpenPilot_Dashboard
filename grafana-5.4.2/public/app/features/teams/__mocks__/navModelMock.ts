export const getMockNavModel = (pageName: string) => {
  return {
    node: {
      active: false,
      icon: 'gicon gicon-team',
      id: `team-${pageName}-2`,
      text: `${pageName}`,
      url: 'org/teams/edit/2/members',
      parentItem: {
        img: '/avatar/b5695b61c91d13e7fa2fe71cfb95de9b',
        id: 'team-2',
        subTitle: 'Manage members & settings',
        url: '',
        text: 'test1',
        breadcrumbs: [{ title: 'Teams', url: 'org/teams' }],
        children: [
          {
            active: false,
            icon: 'gicon gicon-team',
            id: 'team-members-2',
            text: 'Members',
            url: 'org/teams/edit/2/members',
          },
          {
            active: false,
            icon: 'fa fa-fw fa-sliders',
            id: 'team-settings-2',
            text: 'Settings',
            url: 'org/teams/edit/2/settings',
          },
        ],
      },
    },
    main: {
      img: '/avatar/b5695b61c91d13e7fa2fe71cfb95de9b',
      id: 'team-2',
      subTitle: 'Manage members & settings',
      url: '',
      text: 'test1',
      breadcrumbs: [{ title: 'Teams', url: 'org/teams' }],
      children: [
        {
          active: true,
          icon: 'gicon gicon-team',
          id: 'team-members-2',
          text: 'Members',
          url: 'org/teams/edit/2/members',
        },
        {
          active: false,
          icon: 'fa fa-fw fa-sliders',
          id: 'team-settings-2',
          text: 'Settings',
          url: 'org/teams/edit/2/settings',
        },
      ],
    },
  };
};
