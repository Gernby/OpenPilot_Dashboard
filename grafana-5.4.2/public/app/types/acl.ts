export enum OrgRole {
  Viewer = 'Viewer',
  Editor = 'Editor',
  Admin = 'Admin',
}

export interface DashboardAclDTO {
  id?: number;
  dashboardId?: number;
  userId?: number;
  userLogin?: string;
  userEmail?: string;
  teamId?: number;
  team?: string;
  permission?: PermissionLevel;
  role?: OrgRole;
  icon?: string;
  inherited?: boolean;
}

export interface DashboardAclUpdateDTO {
  userId: number;
  teamId: number;
  role: OrgRole;
  permission: PermissionLevel;
}

export interface DashboardAcl {
  id?: number;
  dashboardId?: number;
  userId?: number;
  userLogin?: string;
  userEmail?: string;
  teamId?: number;
  team?: string;
  permission?: PermissionLevel;
  role?: OrgRole;
  icon?: string;
  name?: string;
  inherited?: boolean;
  sortRank?: number;
}

export interface DashboardPermissionInfo {
  value: PermissionLevel;
  label: string;
  description: string;
}

export interface NewDashboardAclItem {
  teamId: number;
  userId: number;
  role?: OrgRole;
  permission: PermissionLevel;
  type: AclTarget;
}

export enum PermissionLevel {
  View = 1,
  Edit = 2,
  Admin = 4,
}

export enum DataSourcePermissionLevel {
  Query = 1,
  Admin = 2,
}

export enum AclTarget {
  Team = 'Team',
  User = 'User',
  Viewer = 'Viewer',
  Editor = 'Editor',
}

export interface AclTargetInfo {
  value: AclTarget;
  text: string;
}

export const dataSourceAclLevels = [
  { value: DataSourcePermissionLevel.Query, label: 'Query', description: 'Can query data source.' },
];

export const dashboardAclTargets: AclTargetInfo[] = [
  { value: AclTarget.Team, text: 'Team' },
  { value: AclTarget.User, text: 'User' },
  { value: AclTarget.Viewer, text: 'Everyone With Viewer Role' },
  { value: AclTarget.Editor, text: 'Everyone With Editor Role' },
];

export const dashboardPermissionLevels: DashboardPermissionInfo[] = [
  { value: PermissionLevel.View, label: 'View', description: 'Can view dashboards.' },
  { value: PermissionLevel.Edit, label: 'Edit', description: 'Can add, edit and delete dashboards.' },
  {
    value: PermissionLevel.Admin,
    label: 'Admin',
    description: 'Can add/remove permissions and can add, edit and delete dashboards.',
  },
];
