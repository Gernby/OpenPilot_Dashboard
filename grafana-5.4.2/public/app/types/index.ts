import { Team, TeamsState, TeamState, TeamGroup, TeamMember } from './teams';
import { AlertRuleDTO, AlertRule, AlertRulesState } from './alerting';
import { LocationState, LocationUpdate, UrlQueryMap, UrlQueryValue } from './location';
import { NavModel, NavModelItem, NavIndex } from './navModel';
import { FolderDTO, FolderState, FolderInfo } from './folders';
import { DashboardState } from './dashboard';
import { DashboardAcl, OrgRole, PermissionLevel } from './acl';
import { ApiKey, ApiKeysState, NewApiKey } from './apiKeys';
import { Invitee, OrgUser, User, UsersState, UserState } from './user';
import { DataSource, DataSourcesState } from './datasources';
import {
  TimeRange,
  LoadingState,
  TimeSeries,
  TimeSeriesVM,
  TimeSeriesVMs,
  TimeSeriesStats,
  NullValueMode,
  DataQuery,
  DataQueryResponse,
  DataQueryOptions,
} from './series';
import { PanelProps, PanelOptionsProps } from './panel';
import { PluginDashboard, PluginMeta, Plugin, PluginsState } from './plugins';
import { Organization, OrganizationState } from './organization';
import {
  AppNotification,
  AppNotificationSeverity,
  AppNotificationsState,
  AppNotificationTimeout,
} from './appNotifications';
import { DashboardSearchHit } from './search';

export {
  Team,
  TeamsState,
  TeamState,
  TeamGroup,
  TeamMember,
  AlertRuleDTO,
  AlertRule,
  AlertRulesState,
  LocationState,
  LocationUpdate,
  NavModel,
  NavModelItem,
  NavIndex,
  UrlQueryMap,
  UrlQueryValue,
  FolderDTO,
  FolderState,
  FolderInfo,
  DashboardState,
  DashboardAcl,
  OrgRole,
  PermissionLevel,
  DataSource,
  PluginMeta,
  ApiKey,
  ApiKeysState,
  NewApiKey,
  Plugin,
  PluginsState,
  DataSourcesState,
  Invitee,
  OrgUser,
  User,
  UsersState,
  TimeRange,
  LoadingState,
  PanelProps,
  PanelOptionsProps,
  TimeSeries,
  TimeSeriesVM,
  TimeSeriesVMs,
  NullValueMode,
  TimeSeriesStats,
  DataQuery,
  DataQueryResponse,
  DataQueryOptions,
  PluginDashboard,
  Organization,
  OrganizationState,
  AppNotification,
  AppNotificationsState,
  AppNotificationSeverity,
  AppNotificationTimeout,
  DashboardSearchHit,
  UserState,
};

export interface StoreState {
  navIndex: NavIndex;
  location: LocationState;
  alertRules: AlertRulesState;
  teams: TeamsState;
  team: TeamState;
  folder: FolderState;
  dashboard: DashboardState;
  dataSources: DataSourcesState;
  users: UsersState;
  organization: OrganizationState;
  appNotifications: AppNotificationsState;
  user: UserState;
}
