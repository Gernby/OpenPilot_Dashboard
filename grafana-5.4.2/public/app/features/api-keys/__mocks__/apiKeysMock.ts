import { ApiKey, OrgRole } from 'app/types';

export const getMultipleMockKeys = (numberOfKeys: number): ApiKey[] => {
  const keys: ApiKey[] = [];
  for (let i = 1; i <= numberOfKeys; i++) {
    keys.push({
      id: i,
      name: `test-${i}`,
      role: OrgRole.Viewer,
    });
  }

  return keys;
};

export const getMockKey = (): ApiKey => {
  return {
    id: 1,
    name: 'test',
    role: OrgRole.Admin,
  };
};
