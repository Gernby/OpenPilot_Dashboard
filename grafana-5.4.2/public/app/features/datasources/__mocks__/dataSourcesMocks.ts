import { DataSource } from 'app/types';

export const getMockDataSources = (amount: number): DataSource[] => {
  const dataSources = [];

  for (let i = 0; i <= amount; i++) {
    dataSources.push({
      access: '',
      basicAuth: false,
      database: `database-${i}`,
      id: i,
      isDefault: false,
      jsonData: { authType: 'credentials', defaultRegion: 'eu-west-2' },
      name: `dataSource-${i}`,
      orgId: 1,
      password: '',
      readOnly: false,
      type: 'cloudwatch',
      typeLogoUrl: 'public/app/plugins/datasource/cloudwatch/img/amazon-web-services.png',
      url: '',
      user: '',
    });
  }

  return dataSources;
};

export const getMockDataSource = (): DataSource => {
  return {
    access: '',
    basicAuth: false,
    database: '',
    id: 13,
    isDefault: false,
    jsonData: { authType: 'credentials', defaultRegion: 'eu-west-2' },
    name: 'gdev-cloudwatch',
    orgId: 1,
    password: '',
    readOnly: false,
    type: 'cloudwatch',
    typeLogoUrl: 'public/app/plugins/datasource/cloudwatch/img/amazon-web-services.png',
    url: '',
    user: '',
  };
};
