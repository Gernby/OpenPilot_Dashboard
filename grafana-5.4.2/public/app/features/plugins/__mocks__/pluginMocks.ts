import { Plugin } from 'app/types';

export const getMockPlugins = (amount: number): Plugin[] => {
  const plugins = [];

  for (let i = 0; i <= amount; i++) {
    plugins.push({
      defaultNavUrl: 'some/url',
      enabled: false,
      hasUpdate: false,
      id: `${i}`,
      info: {
        author: {
          name: 'Grafana Labs',
          url: 'url/to/GrafanaLabs',
        },
        description: 'pretty decent plugin',
        links: ['one link'],
        logos: { small: 'small/logo', large: 'large/logo' },
        screenshots: `screenshot/${i}`,
        updated: '2018-09-26',
        version: '1',
      },
      latestVersion: `1.${i}`,
      name: `pretty cool plugin-${i}`,
      pinned: false,
      state: '',
      type: '',
    });
  }

  return plugins;
};

export const getMockPlugin = () => {
  return {
    defaultNavUrl: 'some/url',
    enabled: false,
    hasUpdate: false,
    id: '1',
    info: {
      author: {
        name: 'Grafana Labs',
        url: 'url/to/GrafanaLabs',
      },
      description: 'pretty decent plugin',
      links: ['one link'],
      logos: { small: 'small/logo', large: 'large/logo' },
      screenshots: 'screenshot/1',
      updated: '2018-09-26',
      version: '1',
    },
    latestVersion: '1',
    name: 'pretty cool plugin 1',
    pinned: false,
    state: '',
    type: '',
  };
};
