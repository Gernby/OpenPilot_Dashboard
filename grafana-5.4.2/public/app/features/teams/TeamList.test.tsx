import React from 'react';
import { shallow } from 'enzyme';
import { Props, TeamList } from './TeamList';
import { NavModel, Team } from '../../types';
import { getMockTeam, getMultipleMockTeams } from './__mocks__/teamMocks';

const setup = (propOverrides?: object) => {
  const props: Props = {
    navModel: {} as NavModel,
    teams: [] as Team[],
    loadTeams: jest.fn(),
    deleteTeam: jest.fn(),
    setSearchQuery: jest.fn(),
    searchQuery: '',
    teamsCount: 0,
    hasFetched: false,
  };

  Object.assign(props, propOverrides);

  const wrapper = shallow(<TeamList {...props} />);
  const instance = wrapper.instance() as TeamList;

  return {
    wrapper,
    instance,
  };
};

describe('Render', () => {
  it('should render component', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });

  it('should render teams table', () => {
    const { wrapper } = setup({
      teams: getMultipleMockTeams(5),
      teamsCount: 5,
      hasFetched: true,
    });

    expect(wrapper).toMatchSnapshot();
  });
});

describe('Life cycle', () => {
  it('should call loadTeams', () => {
    const { instance } = setup();

    instance.componentDidMount();

    expect(instance.props.loadTeams).toHaveBeenCalled();
  });
});

describe('Functions', () => {
  describe('Delete team', () => {
    it('should call delete team', () => {
      const { instance } = setup();
      instance.deleteTeam(getMockTeam());

      expect(instance.props.deleteTeam).toHaveBeenCalledWith(1);
    });
  });

  describe('on search query change', () => {
    it('should call setSearchQuery', () => {
      const { instance } = setup();
      const mockEvent = { target: { value: 'test' } };

      instance.onSearchQueryChange(mockEvent);

      expect(instance.props.setSearchQuery).toHaveBeenCalledWith('test');
    });
  });
});
