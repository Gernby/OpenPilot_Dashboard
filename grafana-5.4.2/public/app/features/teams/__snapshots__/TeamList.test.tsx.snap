// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render should render component 1`] = `
<div>
  <PageHeader
    model={Object {}}
  />
  <PageLoader
    pageName="Teams"
  />
</div>
`;

exports[`Render should render teams table 1`] = `
<div>
  <PageHeader
    model={Object {}}
  />
  <div
    className="page-container page-body"
  >
    <div
      className="page-action-bar"
    >
      <div
        className="gf-form gf-form--grow"
      >
        <label
          className="gf-form--has-input-icon gf-form--grow"
        >
          <input
            className="gf-form-input"
            onChange={[Function]}
            placeholder="Search teams"
            type="text"
            value=""
          />
          <i
            className="gf-form-input-icon fa fa-search"
          />
        </label>
      </div>
      <div
        className="page-action-bar__spacer"
      />
      <a
        className="btn btn-success"
        href="org/teams/new"
      >
        New team
      </a>
    </div>
    <div
      className="admin-list-table"
    >
      <table
        className="filter-table filter-table--hover form-inline"
      >
        <thead>
          <tr>
            <th />
            <th>
              Name
            </th>
            <th>
              Email
            </th>
            <th>
              Members
            </th>
            <th
              style={
                Object {
                  "width": "1%",
                }
              }
            />
          </tr>
        </thead>
        <tbody>
          <tr
            key="1"
          >
            <td
              className="width-4 text-center link-td"
            >
              <a
                href="org/teams/edit/1"
              >
                <img
                  className="filter-table__avatar"
                  src="some/url/"
                />
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/1"
              >
                test-1
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/1"
              >
                test-1@test.com
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/1"
              >
                1
              </a>
            </td>
            <td
              className="text-right"
            >
              <DeleteButton
                onConfirmDelete={[Function]}
              />
            </td>
          </tr>
          <tr
            key="2"
          >
            <td
              className="width-4 text-center link-td"
            >
              <a
                href="org/teams/edit/2"
              >
                <img
                  className="filter-table__avatar"
                  src="some/url/"
                />
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/2"
              >
                test-2
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/2"
              >
                test-2@test.com
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/2"
              >
                2
              </a>
            </td>
            <td
              className="text-right"
            >
              <DeleteButton
                onConfirmDelete={[Function]}
              />
            </td>
          </tr>
          <tr
            key="3"
          >
            <td
              className="width-4 text-center link-td"
            >
              <a
                href="org/teams/edit/3"
              >
                <img
                  className="filter-table__avatar"
                  src="some/url/"
                />
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/3"
              >
                test-3
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/3"
              >
                test-3@test.com
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/3"
              >
                3
              </a>
            </td>
            <td
              className="text-right"
            >
              <DeleteButton
                onConfirmDelete={[Function]}
              />
            </td>
          </tr>
          <tr
            key="4"
          >
            <td
              className="width-4 text-center link-td"
            >
              <a
                href="org/teams/edit/4"
              >
                <img
                  className="filter-table__avatar"
                  src="some/url/"
                />
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/4"
              >
                test-4
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/4"
              >
                test-4@test.com
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/4"
              >
                4
              </a>
            </td>
            <td
              className="text-right"
            >
              <DeleteButton
                onConfirmDelete={[Function]}
              />
            </td>
          </tr>
          <tr
            key="5"
          >
            <td
              className="width-4 text-center link-td"
            >
              <a
                href="org/teams/edit/5"
              >
                <img
                  className="filter-table__avatar"
                  src="some/url/"
                />
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/5"
              >
                test-5
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/5"
              >
                test-5@test.com
              </a>
            </td>
            <td
              className="link-td"
            >
              <a
                href="org/teams/edit/5"
              >
                5
              </a>
            </td>
            <td
              className="text-right"
            >
              <DeleteButton
                onConfirmDelete={[Function]}
              />
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
